### PR TITLE
Updated standalone installation requirements

### PR DESCRIPTION
**Pull request by bbfrederick**
## Changes proposed in this pull request
This PR updates the external dependencies listed in installation.rst for standalone installation to match the packages and versions found in the Dockerfile.
There is also a typo fixed in CONTRIBUTING.md

This should close issue #2008, which I raised today.

The need for an updated version of ICA-AROMA is pretty clear, as fmriprep now uses a flag that doesn't exist in the older version.  I used the version and download link used in the Dockerfile - if there's a more appropriate permalink that might be better.
The bids-validator version has been updated to match the Dockerfile.
I can't see any indication that c3d is installed in the Dockerfile, so I removed it as a dependency.

## Documentation that should be reviewed

This is a documentation only fix, which modifies docs/installation.rst and CONTRIBUTING.md

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -144,10 +144,9 @@ the ``fmriprep`` package:
 - FSL_ (version 5.0.9)
 - ANTs_ (version 2.2.0 - NeuroDocker build)
 - AFNI_ (version Debian-16.2.07)
-- `C3D <https://sourceforge.net/projects/c3d/>`_ (version 1.0.0)
 - FreeSurfer_ (version 6.0.1)
-- `ICA-AROMA <https://github.com/rhr-pruim/ICA-AROMA/>`_ (version 0.4.1-beta)
-- `bids-validator <https://github.com/bids-standard/bids-validator>`_ (version 1.1.0)
+- `ICA-AROMA <https://github.com/oesteban/ICA-AROMA/archive/v0.4.5.tar.gz>`_ (version 0.4.5)
```
Comment (effigies) [inline docs/installation.rst]: Ugh. I didn't realize we have an unofficial release in here.

@oesteban I see @maartenmennes merged https://github.com/maartenmennes/ICA-AROMA/pull/43. Perhaps we should pin:

```suggestion
- `ICA-AROMA <https://github.com/maartenmennes/ICA-AROMA/archive/e8d7a58.tar.gz>`_ (commit e8d7a58, post v0.4.4-beta)
```

WDYT?

Comment (bbfrederick) [reply docs/installation.rst]: Seems reasonable to me.

Comment (bbfrederick) [reply docs/installation.rst]: Well, my reading of the Dockerfile was not in fact as careful as I thought it was.  C3D _is_ installed when other packages are pulled from Neurodebian (as convert3d).  So the line:

- `C3D <https://sourceforge.net/projects/c3d/>`_ (version 1.0.0)

Should _NOT_ be removed from the standalone dependencies.  Should I open a new PR, or can this one be fixed?

Comment (effigies) [reply docs/installation.rst]: This one can be fixed. Just push to the same branch and it'll update.

**Pull request merged**